index.html

```diff
@@ -61,9 +73,36 @@ <h1 id="header-title"> Eloquent UI </h3>
         <div class="load-list">
             <!-- List Component -->
         </div>
+        <div class="load-navigation">
+            <!-- Navigation Component -->
+        </div>
+        <div class="load-modal">
+            <!-- Modal Component -->
+        </div>
+        <div class="load-rating">
+            <!-- Rating Component -->
+        </div>
+        <div class="load-slider">
+            <!-- Slider Component -->
+        </div>
+        <div class="load-typography">
+            <!-- Typography Component -->
+        </div>
     </main>
-    <footer></footer>
+    <footer>
+        <div class="footer-heder">Proudly made in <img src="/images/india.png" class="img-footer"> by Suraj Gupta <span
+                style="font-weight: 100;">currently V1.0</span></div>
+        <ul class="list-items-footer">
+            <li class="footer-list-icon"><a href="https://www.linkedin.com/in/suraj-gupta-a02463139/" target="_blank"><i
+                        class="fa fa-linkedin-square" aria-hidden="true"></i></a></li>
+            <li class="footer-list-icon"><a href="https://github.com/SurajGupta07" target="_blank"><i
+                        class="fa fa-github" aria-hidden="true"></i></a></li>
+            <li class="footer-list-icon"><a href='https://www.instagram.com/suraj_gupta_07_/' target="_blank"><i
+                        class="fa fa-instagram" aria-hidden="true"></i></a></li>
+        </ul>
+    </footer>
```

Comment:
Footer elements are out of sync, appearing below the footer and there is margin at the bottom of the footer